dotcom-rendering/scripts/webpack/webpack.config.js

```diff
@@ -114,18 +114,27 @@ module.exports = [
 						platform: 'browser.legacy',
 					}),
 					require(`./webpack.config.browser`)({
-						isLegacyJS: true,
+						bundle: 'legacy',
 						sessionId,
 					}),
 				),
 		  ]
 		: []),
 	merge(
 		commonConfigs({
-			platform: 'browser',
+			platform: 'browser.modern',
+		}),
+		require(`./webpack.config.browser`)({
+			bundle: 'modern',
+			sessionId,
+		}),
+	),
+	merge(
+		commonConfigs({
+			platform: 'browser.variant',
 		}),
 		require(`./webpack.config.browser`)({
-			isLegacyJS: false,
+			bundle: 'variant',
 			sessionId,
 		}),
```

Comment:
Should we _always_ be building the variant webpack, or should this be something that is only enabled when we have a test we want to run? I'm thinking that this will increase build times, etc.

Reply:
I don’t think there is another way to do this, as the building is indepedent from the test bucketing.

Reply:
If we only 'enable' the variant build in the webpack config when we have a test to run, then in the request logic it should be relatively simple to try and 'read' in the variant manifest, and if it doesn't exist we should fall back to the modern/legacy variants.

Reply:
I have looked at implementing this in bcabdd463d95de4ab3425281ea1be9dbbf258af8. I guess the key value is to make sure that we want the ability to build variant bundles in the future.

We have to make sure not to muddy the waters of the results, because it will be possible for a reader to be in the variant bucker of the [frontend#25443](https://github.com/guardian/frontend/pull/25443) experiment, but not be served the variant because we have not built it in DCR.